boards/arm/mec172xevb_assy6906/pinmux.c

```diff
@@ -80,6 +80,61 @@ static void brd_cfg_uart(struct pinmux_ports_t *pp)
 #endif
 }
 
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(spi0), okay)
+
+#if DT_PROP(DT_INST(0, microchip_xec_qmspi), port_sel) == 0
+/* shared QMSPI port: Two chip selects, single, dual, or quad */
+static void brd_cfg_qmspi(struct pinmux_ports_t *pp)
+{
+#if DT_PROP(DT_INST(0, microchip_xec_qmspi), chip_select) == 0
```

Comment:
can the gpio port/pin be retreived from device tree so we don't need this ifdef?